diff --git a/BlackHole/BlackHole.c b/BlackHole/BlackHole.c
index 4fb2788..a14a00b 100644
--- a/BlackHole/BlackHole.c
+++ b/BlackHole/BlackHole.c
@@ -777,7 +777,7 @@ static OSStatus	BlackHole_Initialize(AudioServerPlugInDriverRef inDriver, AudioS
 	//	set the box name directly as a last resort
 	if(gBox_Name == NULL)
 	{
-		gBox_Name = CFSTR("BlackHole Box");
+		gBox_Name = CFSTR("AudioBridge Box");
 	}
 	
 	//	calculate the host ticks per frame
@@ -1909,14 +1909,14 @@ static OSStatus	BlackHole_GetBoxPropertyData(AudioServerPlugInDriverRef inDriver
 		case kAudioObjectPropertyModelName:
 			//	This is the human readable name of the maker of the box.
 			FailWithAction(inDataSize < sizeof(CFStringRef), theAnswer = kAudioHardwareBadPropertySizeError, Done, "BlackHole_GetBoxPropertyData: not enough space for the return value of kAudioObjectPropertyManufacturer for the box");
-			*((CFStringRef*)outData) = CFSTR("BlackHole");
+			*((CFStringRef*)outData) = CFSTR("AudioBridge");
 			*outDataSize = sizeof(CFStringRef);
 			break;
-			
+
 		case kAudioObjectPropertyManufacturer:
 			//	This is the human readable name of the maker of the box.
 			FailWithAction(inDataSize < sizeof(CFStringRef), theAnswer = kAudioHardwareBadPropertySizeError, Done, "BlackHole_GetBoxPropertyData: not enough space for the return value of kAudioObjectPropertyManufacturer for the box");
-			*((CFStringRef*)outData) = CFSTR("Existential Audio Inc.");
+			*((CFStringRef*)outData) = CFSTR("PUBLIC WORKS");
 			*outDataSize = sizeof(CFStringRef);
 			break;
 			
@@ -1935,7 +1935,7 @@ static OSStatus	BlackHole_GetBoxPropertyData(AudioServerPlugInDriverRef inDriver
 		case kAudioObjectPropertySerialNumber:
 			//	This is the human readable serial number of the box.
 			FailWithAction(inDataSize < sizeof(CFStringRef), theAnswer = kAudioHardwareBadPropertySizeError, Done, "BlackHole_GetBoxPropertyData: not enough space for the return value of kAudioObjectPropertySerialNumber for the box");
-			*((CFStringRef*)outData) = CFSTR("dd658747-4b9a-4de8-a001-c6a2ef1bb235");
+			*((CFStringRef*)outData) = CFSTR("5edc9305-0d9c-4f39-a901-8757bbf07f9d");
 			*outDataSize = sizeof(CFStringRef);
 			break;
 			
